ui/lib/core/addon/components/ttl-picker2.js

```diff
@@ -85,6 +90,11 @@ export default TtlForm.extend({
         unit = 'm';
       }
       time = convertFromSeconds(value, unit);
+
+      // make sure what is displayed is also set on the model.
+      // for example, if using OpenAPI and you don't set a defaultValue, OpenAPI sends 30.
+      // Yet TTL is displaying 30s. Update the state to match here.
+      next(() => this.handleChange());
```

Comment:
`30` and `30s` are interchangeable, so I don't think we should need to trigger an onChange here